packages/web-components/fast-components/src/divider/divider.styles.ts

```diff
@@ -22,7 +22,14 @@ export const dividerStyles: (
             box-sizing: content-box;
             height: 0;
             margin: calc(${designUnit} * 1px) 0;
-            border: none;
             border-top: calc(${strokeWidth} * 1px) solid ${neutralStrokeDividerRest};
+            border-left: none;
+        }
+
+        :host([orientation="vertical"]) {
+            height: 100%;
+            margin: 0 calc(${designUnit} * 1px);
```

Comment:
Is margin still necessary on either of these?

Reply:
Yes, We discussed this in related conversations to #5407